daml-lf/archive/src/main/scala/com/digitalasset/daml/lf/archive/DecodeV1.scala

```diff
@@ -32,7 +34,7 @@ private[archive] class DecodeV1(minor: LV.Minor) extends Decode.OfPackage[PLF.Pa
       onlySerializableDataDefs: Boolean
   ): Package = {
     val internedStrings: ImmArraySeq[String] = ImmArraySeq(
-      lfPackage.getInternedStringsList.asScala: _*)
+      lfPackage.getInternedStringsList.asScala.toSeq: _*)
```

Comment:
`Seq` is now an alias for `collection.immutable.Seq` and with that varargs also changed.